### ISSUE TITLE
Sessions Cookie is not expired after browser close

### ISSUE THREAD

**Issue by BonBonSlick** [js-cookie/js-cookie#481]
```
import * as Cookie from 'js-cookie';
        Cookie.set('session', 'test');
```
// cookies created
// close browser
// open browser, cookie exists

According to docs

> expires
> Define when the cookie will be removed. Value can be a Number which will be interpreted as days from time of creation or a Date instance. If omitted, the cookie becomes a session cookie.
> 

Cookie set once when form submitted, so after browser reopen it will not be set. Is this expected behaviour?

**Comment by carhartl** [comment]
This is not expected behavior. Could you let us know which browser you‘re using, whether you observe the same behavior in a different browser, and which extensions you‘re using?

**Comment by jakiefeng** [comment]
I encounter the issue too, using chrome 70.

**Comment by carhartl** [comment]
I found this: https://stackoverflow.com/questions/10617954/chrome-doesnt-delete-session-cookies

Do you have “Continue where I left off” activated?

**Comment by jakiefeng** [comment]
> I found this: https://stackoverflow.com/questions/10617954/chrome-doesnt-delete-session-cookies
> 
> Do you have “Continue where I left off” activated?

So that explains it. Thx!

**Comment by BonBonSlick** [comment]
I think yes, it should be documented, or mentioned to avoid future questions like this.

**Comment by KlausHartl4242** [comment]
+1

I think it's worth it mentioning in the faq or readme...

**Comment by ismoil793** [comment]
> I found this: https://stackoverflow.com/questions/10617954/chrome-doesnt-delete-session-cookies
> 
> Do you have “Continue where I left off” activated?

Yes I have this feature turned on, thank you for reporting this chrome feature
